src/pages/en/tutorial/2-pages/2.mdx

```diff
@@ -73,6 +73,10 @@ Now that you have built pages using `.astro` files, let's make some blog posts u
 
 3. Use your browser's Dev Tools to inspect this page. Notice that although you have not typed any HTML elements, your Markdown has been converted to HTML. You can see elements such as headings, paragraphs, and list items.
 
+:::note
+The information at the top of the file, inside the code fences, is called frontmatter. This data—including tags and a post image—won't appear until we access it later in the tutorial. 
```

Comment:
```suggestion
The information at the top of the file, inside the code fences, is called frontmatter. This data—including tags and a post image—is information *about* your post that Astro can use. It does not appear directly on the page, but we will access it later in the tutorial to enhance your site. 
```
How about something like this?

Reply:
Yes! One change:

> It does not appear on the page automatically, but we will access it later in the tutorial to enhance your site

(it will eventually appear directly on the page)

Reply:
Amended my suggestion!